src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/xlf/BuildPropertyPage.xaml.fr.xlf

```diff
@@ -4,167 +4,167 @@
     <body>
       <trans-unit id="BoolProperty|AllowUnsafeBlocks|DisplayName">
         <source>Allow unsafe code</source>
-        <target state="new">Allow unsafe code</target>
+        <target state="translated">Autoriser le code unsafe</target>
```

Comment:
As a suggestion, I'd say `le code dangereux` or `le code peu sûr`.

Reply:
code unsafe is used for other projects and is approved Termstudio term. We will keep translation in this form